static/app/views/settings/organizationMembers/organizationMembersList.tsx

```diff
@@ -140,9 +148,98 @@ class OrganizationMembersList extends AsyncView<Props, State> {
     this.setState(state => ({invited: {...state.invited, [id]: 'success'}}));
   };
 
+  updateInviteRequest = (id: string, data: Partial<Member>) =>
+    this.setState(state => {
+      const inviteRequests = [...state.inviteRequests];
+      const inviteIndex = inviteRequests.findIndex(request => request.id === id);
+
+      inviteRequests[inviteIndex] = {...inviteRequests[inviteIndex], ...data};
+
+      return {inviteRequests};
+    });
+
+  removeInviteRequest = (id: string) =>
+    this.setState(state => ({
+      inviteRequests: state.inviteRequests.filter(request => request.id !== id),
+    }));
+
+  handleInviteRequestAction = async ({
+    inviteRequest,
+    method,
+    data,
+    successMessage,
+    errorMessage,
+    eventKey,
+    eventName,
+  }) => {
+    const {params, organization} = this.props;
+
+    this.setState(state => ({
+      inviteRequestBusy: {...state.inviteRequestBusy, [inviteRequest.id]: true},
+    }));
+
+    try {
+      await this.api.requestPromise(
+        `/organizations/${params.orgId}/invite-requests/${inviteRequest.id}/`,
+        {
+          method,
+          data,
+        }
+      );
+
+      this.removeInviteRequest(inviteRequest.id);
+      addSuccessMessage(successMessage);
+      trackAnalyticsEvent({
```

Comment:
@Neo-Zhixing we should utilize the wrapper analytics function here instead of `trackAnalyticsEvent`: https://github.com/getsentry/sentry/blob/d96e6031f4a96e727e46f0930d7166771c875f73/static/app/utils/advancedAnalytics.tsx#L42 

I'm in the process of doing some conversions (see https://github.com/getsentry/getsentry/pull/5947) and when that's done, I am going to mark `trackAnalyticsEvent` as deprecated.